### PR TITLE
HARP-7194: Fixed tiled geojson picking.

### PR DESCRIPTION
**Pull request by amsXYZ**
Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: @here/harp-mapview-decoder/lib/GeoJsonTiler.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class GeoJsonTiler implements ITiler {
                 buffer: 0, // tile buffer on each side
                 lineMetrics: false, // whether to calculate line metrics
                 promoteId: null, // name of a feature property to be promoted to feature.id
-                generateId: false, // whether to generate feature ids. Cannot be used with promoteId
+                generateId: true, // whether to generate feature ids. Cannot be used with promoteId
```
Comment (pit-rpg) [inline @here/harp-mapview-decoder/lib/GeoJsonTiler.ts]: Maybe this value should be taken from OmvDataSourceParameters.gatherFeatureIds, not hardcoded

Comment (amsXYZ) [reply @here/harp-mapview-decoder/lib/GeoJsonTiler.ts]: They're kind of different though, as ``generateId`` generates a unique id for every feature in the data and ``gatherFeatureIds`` fetches an already generated id (which is sometimes shared with even some other datasources).

**Comment by pit-rpg** [comment]
I tested it and it didn't fix the harp-7194 error.
The inspection tab is still displayed for only some of the selected regions.

**Comment by amsXYZ** [comment]
@heremaps-bot retest this please.

**Pull request merged**